docs/release-notes/release-notes-0.19.0.md

```diff
@@ -211,8 +211,13 @@
 `ProofMatureDelta` used in gossip to be configurable via
 `--gossip.announcement-conf`, with a default value of 6.
 
+
+
 ## lncli Updates
 
+* [Fix](https://github.com/lightningnetwork/lnd/pull/9558) Added additional 
```

Comment:
nit: Probably want to replace "Fix" with "Added" so it's a complete sentence?

Reply:
Thanks, I see the pattern now :)